lib/Renderer/StringRenderer.js

```diff
@@ -2,6 +2,8 @@ import {Renderer} from './Renderer';
 import {ValueProxy} from '../Data/ValueProxy';
 import {escape, escapeAttribute} from '../utils';
 
+const VOID_ELEMENTS = ['area', 'base', 'br', 'col', 'command', 'embed', 'hr', 'img', 'input', 'link', 'meta', 'param', 'source']
```

Comment:
Actually this is SELF_CLOSING_TAGS, not VOID_ELEMENTS. This introduces some confusion.
Also, I suggest you simply introduce general solution:
```js
<ANYTAG> === <ANYTAG /> === <ANYTAG></ANYTAG>
```
It seems to me there is no need in checking what a TAG actually is.